commit 3cfc266e7bc75d5d5b94e534487e8e3205116681
Author: Matthew Booth <mbooth@redhat.com>
Date:   Fri Oct 19 10:59:15 2012 +0100

    Fix regression introduced by b41ff38efcf8525d2b335a4870607eb40d78c6c4
    
    Fixes RHBZ#868129
    (cherry picked from commit ea855d875852d14984ceb5cf053e5c811faa1141)

diff --git a/lib/Sys/VirtConvert/Connection/RHEVTarget.pm b/lib/Sys/VirtConvert/Connection/RHEVTarget.pm
index 14d9be2..fc4d005 100644
--- a/lib/Sys/VirtConvert/Connection/RHEVTarget.pm
+++ b/lib/Sys/VirtConvert/Connection/RHEVTarget.pm
@@ -1039,7 +1039,7 @@ sub _disks
         $diske->setAttribute('ovf:actual_size', $usage_gb);
         $diske->setAttribute('ovf:fileRef', $fileref);
         $diske->setAttribute('ovf:parentRef', '');
-        $diske->setAttribute('ovf:vm_snapshot_id', get_uuid());
+        $diske->setAttribute('ovf:vm_snapshot_id', rhev_util::get_uuid());
         $diske->setAttribute('ovf:volume-format', $vol->_get_rhev_format());
         $diske->setAttribute('ovf:volume-type', $vol->_get_rhev_type());
         $diske->setAttribute('ovf:format', 'http://en.wikipedia.org/wiki/Byte');
